src/builder.rs

```diff
@@ -53,17 +60,23 @@ pub enum BundleType {
 }
 
 impl BundleType {
-    /// The default bundle type has all flags enabled, and does not require a bundle to be produced
-    /// if no spends or outputs have been added to the bundle.
-    pub const DEFAULT: BundleType = BundleType::Transactional {
-        flags: Flags::ENABLED,
+    /// The default bundle type has all flags enabled, ZSA disabled, and does not require a bundle
+    /// to be produced.
```

Comment:
_From str4d ([r2602378140](https://github.com/zcash/orchard/pull/471#discussion_r2602378140))_

Undo this comment deletion:
```suggestion
    /// to be produced if no spends or outputs have been added to the bundle.
```
or if it is no longer accurate, modify it to be so instead of removing it.

Reply:
FIXED